### PR TITLE
#165273547: Feature list bookmarks 

### PR DESCRIPTION
**Pull request by bisonlou**
**What does this PR do?**
• Lists articles bookmarked by a user
• Allows a user to remove bookmarked articles
• Allows a user to click and read a bookmarked articles

**Description of Task to be completed?**
• Preset a user with a view listing all their bookmarked articles

**How should this be manually tested?**
[Check out the review app](https://ah-frontend-dojo-pr-39.herokuapp.com/)

**Any background context you want to provide?**
• Users should be able to bookmark articles for later reading

**What are the relevant pivotal tracker stories?**
[165273547](https://www.pivotaltracker.com/story/show/165273547)

### CODE REVIEW

**Review by kalsmic — APPROVED** [review]
Well done, @bisonlou, we now look forward to bookmarking the article.

**Review by nadralia — APPROVED** [review]

**Review by JamesMudidi — APPROVED** [review]
Nice work Innocent Lou.

File: src/components/Profile/Bookmarks.js [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+
+// 3rd party libraries
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+
+// styles
+import './Bookmarks.scss';
+
+// components
```
Comment (llwasampijja) [inline src/components/Profile/Bookmarks.js]: nice work @bisonlou on documenting your work, but just not to confuse anyone who would be building on your code, I believe this is a helper function and not a component

File: src/components/Profile/Bookmarks.js [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+} from 'store/actions/bookmarkActions';
+
+
+export class Bookmarks extends Component {
+  componentDidMount() {
+    const { dispatchFetchBookmarks } = this.props;
+    dispatchFetchBookmarks();
+  }
+
+  remove = (e) => {
```
Comment (llwasampijja) [inline src/components/Profile/Bookmarks.js]: Also, the function name "remove" is not so descriptive of what it is doing. Perhaps you should consider maybe removeBookmark or anything in that realm!

**Review by llwasampijja — CHANGES_REQUESTED** [review]

**Review by llwasampijja — APPROVED** [review]

**Pull request merged**